habitat/tasks/rearrange/rearrange_grasp_manager.py

```diff
@@ -28,7 +27,7 @@ class RearrangeGraspManager:
 
     def __init__(
         self,
-        sim: RearrangeSim,
+        sim,
```

Comment:
Add back a (new?) type?

Reply:
There is a circular dependency between `RearrangeSim` and `RearrangeGraspManager` so I cannot include the type for `RearrangeGraspManager`.